horton/gbasis/cext.pyx

```diff
@@ -1346,6 +1390,25 @@ cdef class GB2NuclearAttractionIntegral(GB2Integral):
         ))
 
 
+cdef class GB2MomentIntegral(GB2Integral):
```

Comment:
This class can be removed because it is not used in the tests (unless I missed something).